install_sct

```diff
@@ -239,6 +239,16 @@ fi
 # Fetch OS type
 if uname -a | grep -i  darwin > /dev/null 2>&1; then
   # OSX
+
+  # Fix for non-English Unicode systems on MAC
+  if [ -z ${LC_ALL} ]; then
+    echo "export LC_ALL=en_US.UTF-8" >> ~/.bash_profile
```

Comment:
i though we agreed to not modify the bash_profile?  how about updating the env variable during install (to make TF happy), and also modify it in the launcher (to have it run during execution of TF-related functions)?

Reply:
done
I am not able to test, struggling with some problems related to pip ...

Reply:
@abtahizadeh changing local language to reproduce the error is extremely easy and takes 5s. Explanation here: https://github.com/Microsoft/vscode/issues/7301